docs/handbook/systems-administration.md

```diff
@@ -233,61 +386,129 @@ perform only these actions and none other.
 
 RBAC was developed to accomplish this.
 
-#### What is RBAC
 
+###### ProFile EXECute: pfexec(1)
+
+| **Tool**  | **Purpose** |
+|-----------|-------------|
+| [pfexec(1)](https://illumos.org/man/1/pfexec)     |  |
+|       | |
+|     |  |
+|          |  |
```

Comment:
When complete, I think this section would be better located directly after the sudo section.

Reply:
TOC is intended to be something like this this:
    Accounts (PR #232)
    Active Directory Integration (this can be moved anywhere I think)
    Software Management (this PR)
    System Monitoring (processes-->clean-up required, then a PR)
    Service Management Facility (remove current section 'Service Management (SMF)' and replace with this PR)
    Virtualisation
    ...